export.py

```diff
@@ -164,10 +164,13 @@ def bpcExport(self):
             #    QMessageBox.warning(self.parent,"System and station names identical", "There are rows where system and station names are identical. Export aborted.")
             #    return
             timescreenshot = datetime.strptime(row[9],"%Y-%m-%dT%H:%M:%S+00:00")
-            if allowedtime > timescreenshot:
-                QMessageBox.warning(self.parent,"Too old for BPC", "You have been using at least one screenshot which is too old. BPC format only allows screenshots younger than 2 hours. Export aborted.")
-                return
-            
+			
```

Comment:
I am using BPC offline for the route optimization. And I am not instantly OCR all screenshots. So in case I have data from last day, I still can export and import to BPC, without messing with the time-stamps of the files.